generators/eslint/index.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+var generators = require('yeoman-generator');
+var extend = require('deep-extend');
+
+module.exports = generators.Base.extend({
+  constructor: function () {
+    generators.Base.apply(this, arguments);
+
+    this.option('babel', {
+      required: false,
+      defaults: false,
+      desc: 'Compile ES2015 using Babel'
```

Comment:
The description doesn't describe what this generator is doing with babel.

This option should be called `es2015`, "Allow es2015 syntax"